src/CheckoutSdk/Marketplace/MarketplaceAccountHolder.cs

```diff
@@ -3,9 +3,13 @@
 
 namespace Checkout.Marketplace
 {
-    public class MarketplaceAccountHolder : AccountHolder
+    public class MarketplaceAccountHolder
```

Comment:
Is it worth it to have this class? It's basically the same as `Common.Four.AccountHolder`